dpsim-models/include/dpsim-models/DP/DP_Ph1_VoltageSource.h

```diff
@@ -10,6 +10,7 @@
 
 #include <dpsim-models/MNASimPowerComp.h>
 #include <dpsim-models/Signal/CosineFMGenerator.h>
+#include <dpsim-models/Signal/DCGenerator.h>
```

Comment:
I find the name "DCGenerator" pretty confusing. Isnt it just a constant value? Can we call it "Constant"?

@m-mirz Whats your opinion on this?

Reply:
Yes, constant is better. Generator is already used for the equipment type.

Reply:
I think not naming it Generator would be quite confusing, because all other classes used for generating the signal of a voltage source are called generator. Constant is also a bad name, because the value generated is not constant at all. Also ConstantGenerator is a pretty weird name.
I'm open to another name if you can think of a good one.

Reply:
> Constant is also a bad name, because the value generated is not constant at all.

Can you elaborate? I thought the purpose is to produce a constant (non oscillating) signal?
Maybe we can simply get rid of it and just use a Sine wave generator using a frequency of 0?

Reply:
no that's too slow. I'm have changed things around a bit so there is no regression for old code.

Reply:
The sine wave generator is too slow.